code/game/machinery/suit_storage_unit.dm

```diff
@@ -254,7 +254,8 @@
 				mob_occupant.adjustFireLoss(rand(20, 36))
 			else
 				mob_occupant.adjustFireLoss(rand(10, 16))
-			mob_occupant.emote("scream")
+			if(iscarbon(mob_occupant))
```

Comment:
```suggestion
			if(iscarbon(mob_occupant) && mob_occupant.stat < UNCONSCIOUS)
```
The dead and the sleeping cannot scream.

Reply:
Good check, thought emotes checked if you qualified to do them.